src/client/driver/driver-link/parent-window.js

```diff
@@ -0,0 +1,37 @@
+import { SetAsMasterMessage } from './messages';
```

Comment:
TypeScript?

Reply:
It's a client code. Need to setup TypeScript compilation for client code in Gulpfile.js.
I guess we can do it the separate PR and rewrite the necessary files in TypeScript later.